conbench/app/_plots.py

```diff
@@ -211,8 +211,23 @@ def _source(
         # the tooltip string labels?
         points = [x.split(" ")[0] for x in means]
 
-    source_data = dict(x=dates, y=points, commits=commits, means=means)
-    return bokeh.models.ColumnDataSource(data=source_data)
+    commit_hashes_short = []
+    for d in data:
+        try:
+            commit_hashes_short.append("#" + d["sha"][:7])
+        except KeyError:
+            # https://github.com/conbench/conbench/issues/589
```

Comment:
As we talked about in this issue, you can probably adjust the code down below that sends a different list of dicts to the `data` input!

Reply:
done, commit pushed